ui/package.json

```diff
@@ -65,7 +65,8 @@
     "@ember/test-waiters": "^3.0.0",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@hashicorp/ember-flight-icons": "^3.0.9",
+    "@hashicorp/design-system-components": "^2.12.2",
```

Comment:
had to upgrade because `@container` support for `<Hds::Copy::Snippet>` was added recently